src/quadtree.js

```diff
@@ -0,0 +1,112 @@
+import AABB from './aabb';
+import Victor from 'victor';
+
+export default class QuadTree {
+    constructor(aabb, maxObjects = 12, maxLevels = 4, level = 0) {
+        this._aabb = aabb;
+        this._maxObjects = maxObjects;
+        this._maxLevels = maxLevels;
+        this._nodeArr = new Array();
+        this._objectArr = new Array();
+        this._level = level;
+    }
+
+    /*clear() {
+        this = new QuadTree(this._aabb, this._maxObjects, this._maxLevels);
+    }*/
+
+    put(obj) {
+        if (this._nodeArr.length > 0) {
+            const index = this._getIndex(obj);
+
+            if (index !== -1) {
+                this._nodeArr[index].put(obj);
+            }
+        }
+
+        this._objectArr.push(obj);
+
+        if (this._objectArr.length > this._maxObjects && this._level < this._maxLevels) {
+            this._split();
+        }
+    }
+
+    get(obj) {
+        const index = this._getIndex(obj);
+        if (index !== -1 && this._nodeArr.length > 0) {
+            return this._nodeArr[index].get(obj);
+        }
+
+        return this._objectArr;
+    }
+
+    remove(obj, idStr) {
+        const index = this._objectArr.findIndex(element => element[idStr] === obj[idStr]);
```

Comment:
Our convention for arrow functions is wrapping the arguments in `()` even if there's only 1 argument.